training/tlt/pedestrian_detection/extract_kitti_labels.py

```diff
@@ -0,0 +1,54 @@
+import os
+import argparse
+import pandas as pd
+from pathlib import Path
+
+
+def extract_labels(args):
+    """
+    This function will create KITTI format annotation from the Oxford Town Center csv annotation file
+    your should specify the csv file path, images width and height.
+    The function will create a `label` directory and for each frame create a KITTI format `txt` file in the directory
+    """
+    gt = pd.read_csv(args.annotation_path, header=None)
+    # The width of original frames are 1920
+    width_ratio = args.image_width / 1920
+    # The height of original frames are 1080
+    height_ratio = args.image_height / 1080
+    annotation_path = Path(args.annotation_path)
+    dataset_dir = annotation_path.parent
+    labels_dir = os.path.join(str(dataset_dir), "labels")
+    if not os.path.exists(labels_dir):
+        os.mkdir(labels_dir)
+
+    def prepare_boxes(bbox):
```

Comment:
maybe rename function to convert_box_from_normalized() ?

Reply:
Since it does not just convert the boxes from normalized and it does clip, scale and change their type to string to prepare them to save in txt files I think  `convert_box_from_normalized()` is not what the function actually do. Do you have any other suggestion?